sml-bin/redeploy-testlib.sml

```diff
@@ -0,0 +1,45 @@
+open OS.Path;
+open OS.FileSys;
+
+fun printLn x = print (x ^ "\n")
+
+fun fileExists x = access (x,[])
+
+fun testLibName base slug =
+    joinDirFile
+        {
+          dir = (concat (base,slug))
+        , file = "testlib.sml"
+        }
+
+fun dirList dirname = let
+    fun go d xs =
+        case readDir d of
+            SOME x => go d (x::xs)
+          | NONE   => (closeDir d; xs);
+in
+    go (openDir dirname) []
+end
+
+fun fileBytes path = let
+    val in' = BinIO.openIn path;
+    val bytes = BinIO.inputAll in';
+in
+    BinIO.closeIn in'; bytes
+end
+
+fun writeFileBytes bytes path = let
+    val out = BinIO.openOut path;
+in
+    BinIO.output (out, bytes);
+    BinIO.closeOut out
+end
+
+val _ = let
+    val concept = concat ((getDir ()), (fromUnixPath "exercise/concept"));
+    val practice = concat ((getDir ()), (fromUnixPath "exercises/practice"));
+
+    val testLib = fileBytes "lib/testlib.sml";
+in
+    app (writeFileBytes testLib o testLibName practice) (dirList practice)
```

Comment:
I think a very short docstring explaining what the script does might be a good idea (maybe at the top of the file). Of course the filename essentially says it but I would still prefer a short docstring. Let me know if you think this would not be helpful.

Reply:
A comment basically restating the name of the script seems pretty redundant to me.

Reply:
OK I am fine with that, I was just a little bit worried if somebody else might wonder what exactly "redeploy" might mean.

Reply:
Hmm, well, I often take for granted that English is my first language, aber meine Meinung nach sollen wir die problemen sprachlicher mehrdeutigkeit der Nicht-Muttersprachlichern beachten :upside_down_face: 

So, maybe it is best for there to be some more direct comments in the script :smile: